pattoo/api/portal/react/src/DatapointRow.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+
+class DatapointRow extends React.Component {
```

Comment:
Where are the comments on what you are doing and why? This is important for future developers.